scripts/west_commands/runners/openocd.py

```diff
@@ -40,7 +41,14 @@ def __init__(self, cfg, pre_init=None, reset_halt_cmd=DEFAULT_OPENOCD_RESET_HALT
                  target_handle=DEFAULT_OPENOCD_TARGET_HANDLE):
         super().__init__(cfg)
 
-        support = path.join(cfg.board_dir, 'support')
+        if not path.exists(cfg.board_dir):
+            # try to find the board support in-tree
+            cfg_board_path = path.normpath(cfg.board_dir)
+            _temp_path = cfg_board_path.split("boards/")[1]
+            support = path.join(environ['ZEPHYR_BASE'], "boards", _temp_path, 'support')
```

Comment:
think about using:
`from zephyr_ext_common import ZEPHYR_BASE`
like in intel_adsp.py.